src/js/common/components/form-elements/GrowableTable.jsx

```diff
@@ -284,7 +283,7 @@ GrowableTable.propTypes = {
   onRowsUpdate: React.PropTypes.func.isRequired,
   path: React.PropTypes.string.isRequired,
   rows: React.PropTypes.array.isRequired,
-  isValidSection: React.PropTypes.func.isRequired,
+  isValidSection: React.PropTypes.func,
```

Comment:
I think this prop should still be required. This is the main validation function for the GrowableTable component and I believe it's always being passed to the components. If you left it and isValidRow off a GrowableTable component, the validation would definitely not work correctly.